### ISSUE TITLE
Very first round of map change (TTT) items are not equipped.

### ISSUE THREAD

**Issue by serfma** [adamdburton/pointshop#196]
For the very first round of a map change, players will not have items equipped. They're actually equipped in the PointShop menu, they're just are not receiving the items when they spawn in. I'm unsure whether this is the same with other gamemodes.

**Comment by smallo92** [comment]
I am experiencing this same issue.

**Comment by Shaknunic** [comment]
Yeah in each round it just changes to the default ttt model and the person needs to change it back manually... Holy, since 2015 with this issue... And i'm reporting back 3 years in the future. just Nice!
